src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/DebugDirectory/DebugDirectoryBuilder.cs

```diff
@@ -116,6 +119,9 @@ public void AddEmbeddedPortablePdbEntry(BlobBuilder debugMetadata, ushort portab
 
         private static int WriteEmbeddedPortablePdbData(BlobBuilder builder, BlobBuilder debugMetadata)
         {
+#if CORERT
+			throw new PlatformNotSupportedException();
```

Comment:
Moved to partial class. Moved entire public api there, so we don't even need the throw.